### PR TITLE
feat: there is a problem withdrawing an isosceles triangle

### PR DESCRIPTION
**Pull request by nammon72**
Look at my code. I have a problem printing

### CODE REVIEW

File: src/isosceles-triangle.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        // print space
+        for(let j = 1; j<= h-i; j++) {
+            shape += " ";
+        }
+        // print star "*"
+        for(let k = 1; k <= 2*i-1; k++) {
+            shape += "*";
+        }
+        // print space
+        for(let j = 1; j<= h-i; j++) {
```
Comment (anhdiepmmk) [inline src/isosceles-triangle.js:13]: unnecessary for loop => remove this loop

File: src/isosceles-triangle.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            shape += "*";
+        }
+        shape += "\n";
+    }
+    return shape;
+}
+drawIsoscelesTriangle(6);
```
Comment (anhdiepmmk) [inline src/isosceles-triangle.js:20]: console.log(drawIsoscelesTriangle(6));

**Review by anhdiepmmk — CHANGES_REQUESTED** [review]
Pls see my comment

File: src/isosceles-triangle.js (hunk 1 above)
Comment (nammon72) [reply src/isosceles-triangle.js]: okay

File: src/isosceles-triangle.js (hunk 2 above)
Comment (nammon72) [reply src/isosceles-triangle.js]: thanks!

**Pull request merged**